tests/mocha/.eslintrc.json

```diff
@@ -1,18 +1,21 @@
 {
+    "parserOptions": {
+      "sourceType": "module"
+    },
     "env": {
         "browser": true,
         "mocha": true
     },
     "globals": {
         "chai": false,
-        "sinon": false,
-        "testHelpers": true
+        "sinon": false
     },
     "rules": {
         "no-unused-vars": ["off"],
         // Allow uncommented helper functions in tests.
         "require-jsdoc": ["off"],
-        "prefer-rest-params": ["off"]
+        "prefer-rest-params": ["off"],
+        "no-invalid-this": ["off"]
```

Comment:
Probably much better to turn this off only where it is necessary.  See e.g. `core/utils/global.js`.